src/test/app-test.ts

```diff
@@ -112,7 +112,18 @@ test('http status codes need to be respected from top to bottom', async (t) => {
 });
 
 test('screenshot is an image', async (t) => {
-  const res = await server.get(`/screenshot/${testBase}basic-script.html`);
+  const res = await server.post(`/screenshot/${testBase}basic-script.html`);
+  t.is(res.status, 200);
+  t.is(res.header['content-type'], 'image/jpeg');
+  t.is(res.body.length, parseInt(res.header['content-length']));
```

Comment:
this could be 0 and still pass, maybe check that it's > some reasonable byte size.

Reply:
Done, its pretty small since the page just has 2 words on it.